### PR TITLE
Create open_redirect_csiro-au.yml

### PR DESCRIPTION
**Pull request by zoomequipd**
# Description

Add coverage for observed open redirect

# Associated samples

- [Sample 1
](https://platform.sublime.security/messages/472a345c019692e30fa1122cef06d65b8e4a90ac51ea6adf73abc527d25c50a1)

## Associated hunts

- [Hunt 1](https://platform.sublime.security/hunts/6a8fbec4-7cb7-472f-aa3e-e34d35d3246e)

### CODE REVIEW

File: detection-rules/open_redirect_csiro-au.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+ame: "Open Redirect: events.csiro.au"
```
Comment (aidenmitchell) [inline detection-rules/open_redirect_csiro-au.yml:1]:
```suggestion
name: "Open Redirect: events.csiro.au"
```

Comment (aidenmitchell) [reply detection-rules/open_redirect_csiro-au.yml:1]: Will probably need to re-testrules it after this fix

**Comment by zoomequipd** [comment]
no telemetry data. Will leave for another week.

**Comment by zoomequipd** [comment]
4 hits in L14 in shared samples.

**Pull request merged**